src/transforms/groupby.js

```diff
@@ -35,14 +35,24 @@ exports.attributes = {
         ].join(' ')
     },
     style: {
```

Comment:
@etpinard @alexcjohnson Comparing this to similar cases, I'm not 100% sure what `_isLinkedToArray` does or whether it's needed here. I've omitted it.